tests/unit/Specification/InPathTest.php

```diff
@@ -25,9 +25,27 @@ class InPathTest extends \PHPUnit_Framework_TestCase
     private $fixture;
 
     /**
-     * Initializes the fixture for this test.
+     * @covers ::__construct
+     * @covers ::isSatisfiedBy
+     * @covers ::<private>
```

Comment:
I recommand putting an `@covers ::<private>` in the class docblock since it makes little sense to add this to each test; nor to specify with each test which private method was covered